### PR TITLE
#27 - ConfigFile for RepoGet and RepoList slices

### PR DESCRIPTION
**Pull request by genryxy**
Part of #28 
Used `ConfigFile` in `ApiRepoGetSlice` and `ApiRepoListSlice`. Also added corresponding tests

### CODE REVIEW

**Comment by olegmoz** [comment]
@olenagerasimova could you please review this PR?

File: src/test/java/com/artipie/management/api/ApiRepoGetSliceTest.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import org.hamcrest.core.AllOf;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+/**
+ * Test for {@link ApiRepoGetSlice}.
+ * @since 0.4
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+final class ApiRepoGetSliceTest {
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/management/api/ApiRepoGetSliceTest.java:49]: @genryxy I'd suggest to add one more test method for the case when config is not found

File: src/test/java/com/artipie/management/api/ApiRepoListSliceTest.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import org.hamcrest.core.AllOf;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test for {@link ApiRepoGetSlice}.
+ * @since 0.4
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+final class ApiRepoListSliceTest {
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/management/api/ApiRepoListSliceTest.java:51]: @genryxy same here - let's add test case for empty user repo

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy please check my comments

**Comment by genryxy** [comment]
@olenagerasimova thanks, added tests

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks, looks good to merge
